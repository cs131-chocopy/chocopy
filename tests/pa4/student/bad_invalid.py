a:[object]=None
i:object=None
a=[1,2,3,True,"1\n2344",[1,2,3]]
for i in a:
	print(i)
