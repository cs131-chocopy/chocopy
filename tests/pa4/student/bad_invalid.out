1
2
3
True
1
2344
Invalid argument
